src/krux/psbt.py

```diff
@@ -71,7 +71,10 @@ def __init__(self, wallet, psbt_data):
         self.policy = None
         for inp in self.psbt.inputs:
             # get policy of the input
-            inp_policy = get_policy(inp, inp.witness_utxo.script_pubkey, xpubs)
+            try:
```

Comment:
I used this when I was debugging and learning about Blue Wallet's legacy wallet creation issue when a xpub was given to it. As I remember, get_policy method couldn't deal with a legacy transaction read attempt and crashed. I think it is better to let it there until further tests, maybe give a more clear error message. Because if more people use Krux with Blue, there will be more people falling into this problem.

Reply:
Can you file an issue for this? I would like to get to the bottom of it when I get time

Reply:
Can you file an issue for this? I would like to get to the bottom of it when I get time